lib/Conversion/TorchToLinalg/Utils.cpp

```diff
@@ -256,3 +282,10 @@ Value torch_to_linalg::createElementwiseLinalgGeneric(
                                  iteratorTypes, bodyBuild)
       .getResult(0);
 }
+
+Value torch_to_linalg::dynamicCast(OpBuilder &b, Location loc, Value tensor) {
```

Comment:
nit: I would call this something else like `removeSizeInformation` to avoid confusion with `std::dynamic_cast`